frontend/src/employee-frontend/components/messages/MessagesPage.tsx

```diff
@@ -148,6 +158,25 @@ export default React.memo(function MessagesPage({
     }
   }
 
+  const getReceivers = useCallback(() => {
+    const person = prefilledRecipientPerson.getOrElse(null)
+    if (person === null) {
+      return receivers
+    }
+    return [
+      {
+        accountId: selectedAccount?.account.id,
+        receivers: [
+          {
+            id: person.id,
+            name: formatPersonName(person, i18n, true),
+            type: 'CITIZEN'
+          }
+        ]
+      }
+    ] as MessageReceiversResponse[]
```

Comment:
`foo as MessageReceiversResponse[]` is not typesafe, because `as` doesn't do any type checking. Instead, it just trusts that you are right.

It would be better to annotate the function return type instead:

```
const getReceivers = useCallback((): MessageReceiversResponse[] | undefined => {
  ...
```

If you do this, you might notice that the returned value is actually not correct...